### PR TITLE
Close claim window

### PR DESCRIPTION
**Pull request by TomBr0wn**
Update the content and remove the add claim button on the claims homepage to reflect the closing of the claim window. 
Ensure UI and content changes cascade through the flow, including claims details screens, etc.

Lucid for referral:
https://lucid.app/lucidspark/e55a976b-bad2-4c0f-843e-2d67dd697e96/edit?view_items=9WDQsdfHsOGg&invitationId=inv_8b275018-bf25-4769-9cb7-28a721d595e4

Not sure about merging this as there's no logic added around the Window dates. Might be better left as a branch

### CODE REVIEW

**Review by olioldfield — APPROVED** [review]
Looks great to me :)

File: app/views/_includes/deadline-banner.njk [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% set deadlineBanner %}
+
+      <p class="govuk-notification-banner__heading">
+        Final closing date for claims: <strong>19 July 2024 by 11.59 pm</strong>.
+      </p>
+
+
+{% endset %}
```
Comment (simonwhatley) [inline app/views/_includes/deadline-banner.njk:8]:
```suggestion
{% set deadlineBanner %}
  <p class="govuk-notification-banner__heading">
    Final closing date for claims: <strong>19 July 2024 by 11.59 pm</strong>
  </p>
{% endset %}
```

File: app/views/support/organisations/claims/list.njk [hunk 2]
```diff
@@ -33,32 +33,50 @@
...
           href: actions.new
         }) }}
-      {% endif %}
+      {% endif %} #}
+
+
 
       {% if claims.length %}
+        {# <p class="govuk-body">
+         <a class="govuk-footer__link" href="mailto:ittmentor.funding@education.gov.uk">Contact the support team</a> if you have a query about claims.
```
Comment (simonwhatley) [inline app/views/support/organisations/claims/list.njk:70]:
```suggestion
         <a class="govuk-link" href="mailto:ittmentor.funding@education.gov.uk">Contact the support team</a> if you have a query about your claims.
```

File: app/views/support/organisations/claims/list.njk [hunk 3]
```diff
@@ -33,32 +33,50 @@
...
+        </p> #}
         {% include "_includes/claims/list.njk" %}
         {% include "_includes/pagination.njk" %}
+
       {% else %}
         <p class="govuk-body">There are no claims for {{ organisation.name if organisation.name else "your organisation" }}.</p>
+        {# <p class="govuk-body">
+         <a class="govuk-footer__link" href="mailto:ittmentor.funding@education.gov.uk">Contact the support team</a> if you have a query about claims.
```
Comment (simonwhatley) [inline app/views/support/organisations/claims/list.njk:78]:
```suggestion
         <a class="govuk-link" href="mailto:ittmentor.funding@education.gov.uk">Contact the support team</a> if you have a query about your claims.
```

**Review by Nitemaeric — APPROVED** [review]

File: app/views/_includes/claims/details.njk [hunk 4]
```diff
@@ -41,7 +41,7 @@
             visuallyHiddenText: "accredited provider"
           }
         ]
-      } if claim.status == "draft"
+      } if claim.status == "draft" and claim.status === "submitted"
```
Comment (simonwhatley) [inline app/views/_includes/claims/details.njk:44]: Should this be `and` or `or`?
```suggestion
      } if claim.status in ["draft", "submitted"]
```

File: app/views/_includes/claims/details.njk [hunk 5]
```diff
@@ -58,7 +58,7 @@
             visuallyHiddenText: "mentors included in the claim"
           }
         ]
-      } if claim.status == "draft"
+      } if claim.status == "draft" and claim.status === "submitted"
```
Comment (simonwhatley) [inline app/views/_includes/claims/details.njk:61]: Should this be `and` or `or`?
```suggestion
      } if claim.status in ["draft", "submitted"]
```

File: app/views/_includes/claims/hours-of-training.njk [hunk 6]
```diff
@@ -13,7 +13,7 @@
           {{ mentor.hours + " hours" if mentor.hours else "Not entered" }}
         {% endif %}
       </dd>
-      {% if claim.status == "draft" %}
+      {% if claim.status == "draft" and claim.status == "submitted" %}
```
Comment (simonwhatley) [inline app/views/_includes/claims/hours-of-training.njk:16]: Should this be `and` or `or`?
```suggestion
      {% if claim.status in ["draft", "submitted"] %}
```

**Review by simonwhatley — COMMENTED** [review]
@TomBr0wn I've made a couple of suggestions. Also, this PR could benefit from the settings functionality you used in the deadline banner work.

File: app/views/_includes/claims/details.njk (hunk 4 above)
Comment (simonwhatley) [reply app/views/_includes/claims/details.njk:44]: This logic suggests that a claim can be edited even after submission. Is that correct?

**Pull request closed without merging**